components/global/meta.tsx

```diff
@@ -21,68 +17,69 @@ declare global {
   }
 }
 
-const getTitle = (title: string, conference: Conference, dates: Dates) =>
-  `${title !== 'Home' ? title + ' - ' : ''}${conference.Name}${
-    !conference.HideDate && !dates.IsComplete ? ` | ${format(conference.StaticDate, 'do MMMM yyyy')}` : ''
-  }`
+const getTitle = (title: string, date: Date, name: string, showDate: boolean) =>
+  `${title !== 'Home' ? title + ' - ' : ''}${name}${showDate ? ` | ${format(date, 'do MMMM yyyy')}` : ''}`
 
 export const Meta: React.FC<MetaArgs> = ({
-  pageUrl,
   pageTitle,
-  instrumentationKey,
   pageDescription,
-  pageImage,
-  conference,
-  dates,
-  googleAnalyticsId,
+  pageImage = '/static/images/logo-2021.png',
 }) => {
+  const { conference, appConfig, dates } = useConfig()
```

Comment:
The props that are removed have been replaced by using the config hook. Saves passing them down